crates/bin/pindexer/src/governance/schema.sql

```diff
@@ -0,0 +1,75 @@
+CREATE TYPE proposal_stage AS ENUM ('VOTING', 'FINISHED');
+CREATE TYPE proposal_status AS ENUM ('VOTING', 'WITHDRAWN', 'FINISHED', 'CLAIMED');
+CREATE TYPE payload_type AS ENUM ('SIGNALING', 'EMERGENCY', 'PARAMETER_CHANGE', 'COMMUNITY_POOL_SPEND', 'UPGRADE_PLAN', 'FREEZE_IBC_CLIENT', 'UNFREEZE_IBC_CLIENT');
+CREATE TYPE vote_type AS ENUM ('ABSTAIN', 'YES', 'NO');
+CREATE TYPE proposal_outcome AS ENUM ('PASSED', 'FAILED', 'SLASHED');
```

Comment:
I don't think it's a good idea to define a new serialization format here, since every consumer of these tables will need to read it. Instead, we should use the serialization format we already have -- all these enums already have JSON representations, and we should use that. Example from the validator set `AppView`:

```
penumbra_mainnet_dst_a=# SELECT DISTINCT validator_state FROM stake_validator_set;
             validator_state
------------------------------------------
 {"state":"VALIDATOR_STATE_ENUM_ACTIVE"}
 {"state":"VALIDATOR_STATE_ENUM_JAILED"}
 {"state":"VALIDATOR_STATE_ENUM_DEFINED"}
(3 rows)
```

Reply:
Let's say a downstream consumer is trying to use the data produced by this `AppView`. How are they supposed to know about any of these enum variants? They have no source of truth and cannot expect to automatically deserialize data into whatever language they're using. Instead, using TEXT-encoded JSON fixes this problem.